source/hardware/controllers/denon_mc7000.rst

```diff
@@ -1,18 +1,7 @@
 Denon MC7000
-============
+------------
```

Comment:
Why did you change the header underlines? This broke the document tree:

![Screenshot from 2020-11-03 12-46-04](https://user-images.githubusercontent.com/1834516/97981528-b7dd3f80-1dd2-11eb-9a73-959e5ad2e47b.png)

Reply:
Never got there to modify anything. So when it changes this was not by purpose. Will correct it.